next.config.js

```diff
@@ -2,26 +2,7 @@ const withSourcebit = require('sourcebit').sourcebitNext();
 
 module.exports = withSourcebit({
     typescript: { ignoreBuildErrors: false },
-    trailingSlash: true,
     devIndicators: {
         autoPrerender: false
-    },
-    eslint: {
-        // Allow production builds to successfully complete even if your project has ESLint errors.
-        ignoreDuringBuilds: true
-    },
-    webpack: (config, { dev }) => {
-        // Tell webpack to ignore watching content files in the content folder.
-        // Otherwise webpack recompiles the app and refreshes the whole page.
-        // Instead, the src/pages/[...slug].js uses the "withRemoteDataUpdates"
-        // function to update the content on the page without refreshing the
-        // whole page
-        config.watchOptions.ignored.push('/content/');
```

Comment:
We actually do need to keep the ignoring, but in the fixed form: 
`config.watchOptions.ignored.push('**/content/'**);`
We can make it into a one-liner comment and remove the rest of the already-commented-out stuff